### PR TITLE
added softhand description

### PR DESCRIPTION
**Pull request by ghost**
Hi Nadia, 

Simon (Haller) and me today created a new model for the Robotino with the Softhand. The soft_hand_description for now lies under squirrel_common; the other changes we did under squirrel_robotino, where we added the new models. Would you mind checking, if everything is fine? We can also have a short skype call, if there remains stuff to be done from my side for a smooth integration.

Thanks!

### CODE REVIEW

File: soft_hand_description/package.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  <version>0.0.0</version>
+  <description>The soft_hand_description package</description>
+
+  <maintainer email="philipp.zech@uibk.ac.at">Philipp Zech</maintainer>
+  <license>GPLv2</license>
+
+  <author>Philipp Zech</author>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
```
Comment (NadiaHG) [inline soft_hand_description/package.xml:13]: please add

```
<run_depend>gazebo_ros</run_depend>
<run_depend >xacro</run_depend>
```

File: soft_hand_description/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+# add_dependencies(lwr_test_node lwr_test_generate_messages_cpp)
+
+## Specify libraries to link a library or executable target against
+# target_link_libraries(lwr_test_node
+#   ${catkin_LIBRARIES}
+# )
+
+#############
+## Install ##
+#############
```
Comment (NadiaHG) [inline soft_hand_description/CMakeLists.txt:85]:
```
please add the install tags:


install(DIRECTORY urdf
    DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
)

install(DIRECTORY meshes
    DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
)

```

**Review by ipa-nhg — CHANGES_REQUESTED** [review]
Add the soft_hand_description package please to the metapackage xml https://github.com/squirrel-project/squirrel_common/blob/indigo_dev/squirrel_common/package.xml

**Review by ipa-nhg — APPROVED** [review]
Thanks. LGTM :+1:

**Pull request merged**